src/types_definition.jl

```diff
@@ -38,6 +38,31 @@ The `SolverParams` currently implemented within RipQP are:
 """
 abstract type SolverParams end
 
+"""
+Type to write the matrix (.mtx format) and the right hand side (.rhs format) of the system to solve at each iteration.
+
+- `write::Bool`: activate/deactivate writing of the system 
```

Comment:
What would be the point of setting `write = false`?

Reply:
`write=false` by default because we do not want to write systems every time we use RipQP

Reply:
Right but why define a `SystemWrite` struct in that case?

Reply:
I want to put it in the `InputConfig` struct because I think it is more convenient than using `kwargs...` or something like this with the `ripqp` function.  Did you have another idea in mind?